tests/platform_tests/api/test_sfp.py

```diff
@@ -176,6 +177,14 @@ def is_xcvr_resettable(self, xcvr_info_dict):
             return False
         return True
 
+    def is_xcvr_support_lpmode(self, xcvr_info_dict):
+        """Returns True if transceiver is support low power mode, False if not supported"""
+        xcvr_type = xcvr_info_dict["type"]
+        ext_identifier = xcvr_info_dict["ext_identifier"]
+        if not "QSFP" in xcvr_type or "Power Class 1" in ext_identifier:
```

Comment:
Can we specify the SFP type (SFP/SFP+) which doesn't support low power mode instead of < not "QSFP" >?

Reply:
Hi @sujinmkang, since 'SFP', 'SFP+' and 'SFP28' are substring of 'QSFP', 'QSFP+', 'QSFP28' and 'QSFP-DD', it will be very complicated to detect the SFP', 'SFP+' and 'SFP28' with 'in' against the xcvr type. what do you say?

Reply:
@sujinmkang would you please feedback?